flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/batch/table/AggregateTest.scala

```diff
@@ -72,4 +73,16 @@ class AggregateTest extends TableTestBase {
 
     util.verifyPlan(resultTable)
   }
+
+  @Test
```

Comment:
Create a new class, e.g. PythonAggregateTest and move the test case there?

Reply:
Yes. Make sense. It will make Python Aggregate test more clear.